guard/tests/utils.rs

```diff
@@ -64,6 +65,37 @@ pub fn get_full_path_for_resource_file(path: &str) -> String {
     return resource.display().to_string();
 }
 
+pub fn replace_home_directory_with_tilde(text: String) -> String {
+    let home_dir_string = match env::var("HOME") {
+        Ok(home_path) => home_path,
+        Err(_) => panic!("HOME variable required for tests!"),
+    };
+
+    text.replace(&home_dir_string, "~")
+}
+
+pub fn replace_path_with_filenames(text: String) -> String {
+    let extensions = ["yaml", "yml", "json"];
+    // pattern to match anything between "~/" and any of the extensions
+    let pattern = format!(
+        r#"~/(?:[\w/\-]+/)?([\w/\-]+\.(?:{}))"#,
+        extensions.join("|")
+    );
+    let re = Regex::new(&pattern).unwrap();
+    // replace the entire match with match group 1 (the file name)
+    let replaced_filenames = re.replace_all(&text, "$1");
+
+    replaced_filenames.to_string()
+}
+
+pub fn sanitize_path(string_to_sanitize: String) -> String {
```

Comment:
Just to confirm my assumptions this function would take something like `/users/foo/bar/repos/cloudformation-guard/guard/resources/validate/some_s3_template.yaml` and convert it to `~/some_s3_template.yaml` is that correct?

Reply:
> `/users/foo/bar/repos/cloudformation-guard/guard/resources/validate/some_s3_template.yaml`
> 
> into
> 
> `some_s3_template.yaml`
> 
> Keeps us from needing to update any of the fixtures. The `~/` replacer is just to make the regex pattern matching less messy. I left a comment in there about why.

sorry, yeah thats exactly what i meant! awesome, love it!